src/EA.Weee.Email/EventHandlers/OrganisationUserRequestCompletedEventHandler.cs

```diff
@@ -1,21 +1,35 @@
 ﻿namespace EA.Weee.Email.EventHandlers
 {
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Threading.Tasks;
     using EA.Prsd.Core.Domain;
     using EA.Weee.Domain.Events;
     using System.Threading.Tasks;
+    using EA.Weee.Domain.Organisation;
 
     public class OrganisationUserRequestCompletedEventHandler : IEventHandler<OrganisationUserRequestCompletedEvent>
     {
         private IWeeeEmailService emailService;
+        private readonly IOrganisationUserRequestEventHandlerDataAccess dataAccess;
 
-        public OrganisationUserRequestCompletedEventHandler(IWeeeEmailService emailService)
+        public OrganisationUserRequestCompletedEventHandler(IOrganisationUserRequestEventHandlerDataAccess dataAccess, IWeeeEmailService emailService)
         {
             this.emailService = emailService;
+            this.dataAccess = dataAccess;
         }
 
         public async Task HandleAsync(OrganisationUserRequestCompletedEvent @event)
         {
-            await emailService.SendOrganisationUserRequestCompleted(@event.OrganisationUser);
+            IEnumerable<OrganisationUser> activeOrganisationUsers = await dataAccess.FetchActiveOrganisationUsers(@event.OrganisationUser.OrganisationId);
```

Comment:
Tests for this to ensure SendOrganisationUserRequestCompleted is called with the correct parameters based on active users